### PR TITLE
DOC change doc plot kinds

### LINKED ISSUE #124: DOC : Write on the website what are the plot kinds available
On : 
https://benchopt.github.io/cli.html#benchopt-plot
user don't know what they can write as `<kinds>`.

### PR DESCRIPTION
**Pull request by ngazagna**
Corrects the `help` of plot kinds and closes #124

### CODE REVIEW

**Comment by agramfort** [comment]
If you write 

closes #124 in pr description so when we merge it will close the issue automatically

**Review by agramfort — APPROVED** [review]

**Review by tomMoral — COMMENTED** [review]
I would like to avoid hard coding the kind of plots in the help directive. could you check if formating with the global variable is possible? if it is too complicated, don't spend hours on this.

**Comment by ngazagna** [comment]
I did this instead : 
https://github.com/ngazagna/benchOpt/blob/6d62edc3f4d38c0e32448b81e3cc551f75293851/benchopt/cli.py#L158-L160
Can we remove the "+" ?

File: benchopt/cli.py [hunk 1]
```diff
@@ -152,7 +155,9 @@ def run(benchmark, solver_names, forced_solvers, dataset_names,
               "folder.")
 @click.option('--kind', '-k', 'kinds',
               multiple=True, show_default=True, type=str,
-              help='Timeout a solver when run for more than <timeout> seconds')
+              help="Specify the type of figure to plot: " +
+              ", ".join("`{0}`".format(k) for k in PLOT_KINDS_NAMES[:-1]) +
```
Comment (tomMoral) [inline benchopt/cli.py:159]: Use f-string when possible:
```suggestion
              ", ".join(f"`{k}`" for k in PLOT_KINDS_NAMES[:-1]) +
```

**Review by tomMoral — COMMENTED** [review]
A small nitpick.

For the formatting without a list, I am super convinced:
Screenshot_20201229-174114

At least put double quotes. but I think a list could be nicer.

File: benchopt/cli.py [hunk 2]
```diff
@@ -19,6 +19,9 @@
 CONTEXT_SETTINGS = dict(help_option_names=['-h', '--help'])
 
 
+PLOT_KINDS_NAMES = list(PLOT_KINDS.keys())
```
Comment (tomMoral) [inline benchopt/cli.py:22]: To make sure the names are all formatted the same (even the last one which is not processed at the same time), you could do:
```suggestion
PLOT_KINDS_NAMES = list(f'`{k}`' for k in PLOT_KINDS)
```
and then use the name directly in `join` and for the last one.

**Comment by ngazagna** [comment]
What we did with @tomMoral 
Screenshot_kind

File: benchopt/cli.py [hunk 3]
```diff
@@ -19,6 +19,9 @@
 CONTEXT_SETTINGS = dict(help_option_names=['-h', '--help'])
 
 
+PLOT_KINDS_NAMES = list(f'`{k}`' for k in PLOT_KINDS)
+
+
```
Comment (tomMoral) [inline benchopt/cli.py:24]:
```suggestion
```

**Review by tomMoral — APPROVED** [review]
A small clean up but other wise, LGTM!

**Comment by tomMoral** [comment]
Thx a lot @ngazagna for taking the time to do this! :fire:

**Pull request merged**